src/Blazor.Auth0.ClientSide/Authentication.cs

```diff
@@ -94,7 +94,7 @@ public static string BuildAuthorizeUrl(AuthorizeOptions buildAuthorizedUrlOption
         /// <param name="uriHelper">A <see cref="IUriHelper"/> param.</param>
         /// <param name="authorizeOptions">A <see cref="AuthorizeOptions"/> param.</param>
         /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
-        public static async Task Authorize(IJSRuntime jsRuntime, IUriHelper uriHelper, AuthorizeOptions authorizeOptions)
+        public static async Task Authorize(IJSRuntime jsRuntime, NavigationManager uriHelper, AuthorizeOptions authorizeOptions)
```

Comment:
Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?